hls4ml/converters/pytorch/core.py

```diff
@@ -1,6 +1,24 @@
+import numpy as np
+
 from hls4ml.converters.pytorch_to_hls import pytorch_handler
 
 
+@pytorch_handler('Constant')
+def parse_constant_layer(operation, layer_name, node):
+    assert 'Constant' in operation
+
+    layer = {}
```

Comment:
So the suggested change here would just be `layer['inputs'] = []`, and then we'd add a check to the `Constant` node to throw an error if the input list is not empty?

Reply:
Here and in qonnx parser. Then the `self.inputs = []` in `Constant` layer could be a check and a warning before setting. This could be better for futureproofing for constant nodes coming from various sources (like optimizers). @jmitrevs what do you think?

Reply:
I have added it here.